lib/engine/game/g_system18/map_britain_customization.rb

```diff
@@ -0,0 +1,497 @@
+              logo: 'System18/PGS',
+              tokens: [0, 40, 100],
+              coordinates: nil,
+              color: '#cd79a7',
+              reservation_color: nil,
+              max_ownership_percent: 60,
+            }
+          )
+          corps.each_with_index do |c, idx|
+            c[:float_percent] = 20
+            c[:always_market_price] = true
+            c[:tokens].append(100)
+            c[:coordinates] = %w[F7 J5 F9 A6 I12 A8][idx]
+            c[:city] = [1, nil, 1, nil, nil, nil][idx]
+          end
+          corps
+        end
+
+        def map_britain_game_cash
+          { 3 => 420, 4 => 315, 5 => 250 }
+        end
+
+        def map_britain_game_cert_limit
+          { 3 => 16, 4 => 12, 5 => 10 }
+        end
+
+        def map_britain_game_capitalization
+          :incremental
+        end
+
+        def map_britain_game_market
+          self.class::MARKET_1D
+        end
+
+        def map_britain_game_trains(trains)
+          # don't use D trains
+          trains.delete(find_train(trains, 'D'))
+          find_train(trains, '4')[:rusts_on] = '8'
+          # udpate quantities
+          find_train(trains, '2')[:num] = 6
+          find_train(trains, '3')[:num] = 4
+          find_train(trains, '4')[:num] = 3
+          find_train(trains, '5')[:num] = 3
+          find_train(trains, '6')[:num] = 2
+          find_train(trains, '8')[:num] = 99
+          trains.append({
+                          name: '4D',
+                          distance: [{ 'nodes' => %w[city offboard town], 'pay' => 4, 'visit' => 99, 'multiplier' => 2 }],
+                          price: 1000,
+                          num: 99,
+                          available_on: '8',
+                        })
+          trains
+        end
+
+        def map_britain_game_phases
+          self.class::S18_INCCAP_PHASES
+        end
+
+        def map_britain_post_game_phases(phases)
+          phases
+        end
+
+        def map_britain_constants
+          redef_const(:CURRENCY_FORMAT_STR, '£%s')
+          redef_const(:TILE_UPGRADES_MUST_USE_MAX_EXITS, %i[unlabeled_cities])
+          redef_const(:TILE_LAYS, [{ lay: true, upgrade: true, cost: 0 }, { lay_replaced: :if_green_upgraded }])
+        end
+
+        def map_britain_setup
+          # add 2 DRG and PHX tokens each to map
+          dragon = corporations.find { |c| c.name == 'DGN' }
+          hexes.find { |h| h.id == 'F7' }.tile.cities[1].place_token(dragon, dragon.tokens[0], free: true)
+          hexes.find { |h| h.id == 'I8' }.tile.cities[0].place_token(dragon, dragon.tokens[1], free: true)
+
+          phoenix = corporations.find { |c| c.name == 'PHX' }
+          hexes.find { |h| h.id == 'F9' }.tile.cities[1].place_token(phoenix, phoenix.tokens[0], free: true)
+          hexes.find { |h| h.id == 'G8' }.tile.cities[0].place_token(phoenix, phoenix.tokens[1], free: true)
+
+          @britain_mines = BRITAIN_MINE_HEXES.map { |h| hex_by_id(h) }
```

Comment:
It might be worth also setting `@scotland_hexes` and `@wales_hexes` variables here, for later use when calculating bonuses.

Reply:
Yes, that makes sense.